package.json

```diff
@@ -62,30 +60,11 @@
   },
   "repository": "adobe/aio-cli-plugin-console",
   "scripts": {
-    "eslint": "eslint src test",
+    "eslint": "eslint .",
     "test": "npm run eslint && npm run unit-tests",
-    "unit-tests": "jest --ci -w=2",
+    "unit-tests": "jest",
```

Comment:
This is the reason the Windows builds are failing. If you restrict the number of workers to 2 (like the original line), they pass. Revert this line to make the build pass on Windows.

Reply:
See #40 where I validate this